src/continuum_absorption/absorption_H.jl

```diff
@@ -1,5 +1,5 @@
 using Interpolations: LinearInterpolation, Throw
-using StaticArrays: SA
+using HDF5: h5read
 
 using ..ContinuumAbsorption: hydrogenic_bf_absorption, hydrogenic_ff_absorption, ionization_energies
 
```

Comment:
Can you update line 7 to read `const _H⁻_ion_energy = 0.754204 # eV (value adopted in McLaughlin)`